book/frontend/javascript/2.code_conventions_and_linting.md

```diff
@@ -9,11 +9,11 @@ We try to standardize as much as possible the code we write with two main object
 
 The first objective requires the developer to come up with code that is as explicit and clear as possible, and the reviewer of a PR to enforce it at review time. For both points though, we can delegate much of the grunt work to a linter. To put it more clearly:
 
-There are some things an automated tool can't buy you. For everything else, there's [ESLint](https://github.com/eslint/eslint).
+There are some things an automated tool can't buy you. For everything else, there is [ESLint](https://github.com/eslint/eslint).
 
 [buildo/eslint-config](https://github.com/buildo/eslint-config) is a meticulously hand-crafted ESLint configuration that we use on all our projects. Except a few exceptions (you can always turn on/off rules for specific projects), it should exactly allow what we want and prevent much of the unwanted.
 
-It contains rules for JavaScript, rules specific to react, and some additional rules provided by our own plugins like [no-loops](https://github.com/buildo/eslint-plugin-no-loops) and [no-copy-paste](https://github.com/buildo/eslint-plugin-no-copy-paste-default-export).
+It contains rules for JavaScript, rules specific to react, and some additional rules provided by our own plugins like [no-loops](https://github.com/buildo/eslint-plugin-no-loops) and [no-copy-paste](https://github.com/buildo/eslint-plugin-no-copy-paste-default-export). [TSLint](https://palantir.github.io/tslint/) allows us to apply it also to TypeScript code.
```

Comment:
I'd add a note here, something like ", but we don't have a well maintained shared configuration yet. Stay tuned 😎 "